app/src/components/players/PlayerStatsTable.tsx

```diff
@@ -214,7 +214,7 @@ function getSkillColumnDefinitions(player: Player, showVirtualLevels: boolean):
         return <TableSortButton column={column}>Skill</TableSortButton>;
       },
       cell: ({ row }) => {
-        const isSpecialEHP = player.type !== PlayerType.REGULAR || player.build !== PlayerBuild.MAIN;
+        const isSpecialEHP = hasSpecialEhp(player);
```

Comment:
It should be

`const isSpecialEHP = player.type !== PlayerType.REGULAR || hasSpecialEhp(player)`